### PR TITLE
Update README with new required steps on Unix-Like

### PR DESCRIPTION
**Pull request by pedrinho**
I was not being able to build on linux following the README so I think that this should help other people

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +18,7 @@ On Ubuntu, the following packages are needed to build:
  - libpng12-dev
  - libsdl2-dev
  - make
+ - libtool
```
Comment (pphaneuf) [inline README.md]: libtool is required? I, uh, think I don't have it. :smiley:

File: README.md [hunk 2]
```diff
@@ -54,7 +56,14 @@ Building on Unix-style systems
 
 To compile and install this software on a Unix-like system, do this:
 
-$ make (if you checked out the source code from Subversion)
+If you checked out the source code from GitHub: 
+$ libtoolize --force
+$ aclocal
+$ autoheader
+$ automake --force-missing --add-missing
+$ autoconf
```
Comment (pphaneuf) [inline README.md]: All those steps should just be replaced with `autoreconf -i`.

File: README.md [hunk 3]
```diff
@@ -54,7 +56,14 @@ Building on Unix-style systems
...
+
+And then the tradicional build procedure:
```
Comment (pphaneuf) [inline README.md]: s/tradicional/traditional/

**Comment by pphaneuf** [comment]
BTW, it's not a big deal for me, but you might find it easier if you create a branch (in your repository) to do changes, as you might find it a bit confusing once we make changes to _our_ `master`...

**Comment by pphaneuf** [comment]
Oh, but don't do it for this pull request, otherwise it'll get all confused. :stuck_out_tongue:

File: README.md (hunk 1 above)
Comment (pedrinho) [inline README.md]: Just to "libtoolize --force"... which is not needed

File: README.md (hunk 2 above)
Comment (pedrinho) [inline README.md]: I was not aware of this command... worked like a charm here :)

File: README.md (hunk 3 above)
Comment (pedrinho) [inline README.md]: Fixed. How do I update this pull request with those changes?

**Comment by pphaneuf** [comment]
Could you flatten all this into a single commit? You can do this with these commands:

```
git reset --soft 99dd7f7b
git diff --cached  # check that it looks okay
git commit
```

When you push it to github, you'll have to use the `-f` flag.

**Comment by pphaneuf** [comment]
Pull requests map a feature branch (in this case, the branch named "master" in your repository) to a destination branch (our "master"). All you need to do to update the pull request is push to the feature branch. I have merged it in, thanks!

That's another reason why you should always make a branch to work in, rather than use the "master" branch. If you don't, and put all your work in "master", your pull requests will have everything all jumbled up together...

I've seen this many times, patches or pull requests with many features mixed in together, where the project maintainers wanted most of them, but disagreed about one thing, and so couldn't merge it in. Sending smaller, separate pull requests, allows for accepting all the contributions that are not controversial, while discussion and further improvements are done with the controversial ones. :smiley:

**Pull request merged**